tests/publiccloud/azure_cli.pm

```diff
@@ -46,7 +46,7 @@ sub run {
     assert_script_run("az group create -n $resource_group --tags '$tags'");
 
     # Pint - command line tool to query pint.suse.com to get the current image name
-    my $image_name = script_output("pint microsoft images --active --json | jq -r '.images[] | select( (.urn | contains(\"sles-15-sp3:gen2\")) and (.state == \"active\") and (.environment == \"PublicAzure\")).urn'");
+    my $image_name = script_output("pint microsoft images --active --json | jq -r '.images[] | select( (.urn | contains(\"sles-15-sp4\")) and (.state == \"active\")).urn' | head -n1");
```

Comment:
Can you add a check that `$image_name` is not empty?

Reply:
Done.